indico/web/forms/fields/files.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals, absolute_import
+
+from wtforms import Field
+
+from indico.core.config import Config
+from indico.web.forms.widgets import JinjaWidget
+
+
+class FileField(Field):
+    """A dropzone field"""
+
+    widget = JinjaWidget('forms/dropzone_widget.html')
+
+    default_options = {
+        'max_files': 10,
```

Comment:
I would keep the `max_files` defaulted to 10. It's nice in case you want to configure it. But right now we can just use the new `multiple` option in all the `FileField` cases.